spec/models/etd_spec.rb

```diff
@@ -98,6 +98,7 @@
     it { should respond_to(:description) }
     it { should respond_to(:publisher) }
     it { should respond_to(:degree) }
+    it { should respond_to(:degree_date) }
```

Comment:
@scherztc Should this test also verify that the model responds to :committee_member ?